recipes/cairocffi/dlopen-patch.linux.txt

```diff
@@ -0,0 +1,11 @@
+--- cairocffi/__init__.py	2016-11-16 13:45:03.606537277 +0100
++++ cairocffi/__init__.py.linux	2016-11-16 13:45:38.591975235 +0100
+@@ -43,7 +43,7 @@
+     raise OSError("dlopen() failed to load a library: %s" % ' / '.join(names))
+ 
+ 
+-cairo = dlopen(ffi, 'cairo', 'cairo-2')
++cairo = ffi.dlopen('libcairo.so')
```

Comment:
To ensure you will get libcairo from conda I believe it would be better to do something like

```python
os.path.join(sys.prefix, 'lib', 'libcairo.so')
```

You could also wrap both patches into one with an if clause.